samples/cext-trey-research-csharp/TeamsApp/teamsapp.local.yml

```diff
@@ -0,0 +1,50 @@
+# yaml-language-server: $schema=https://aka.ms/teams-toolkit/v1.2/yaml.schema.json
+# Visit https://aka.ms/teamsfx-v5.0-guide for details on this file
+# Visit https://aka.ms/teamsfx-actions for details on actions
+version: v1.2
+
+additionalMetadata:
+  sampleTag: Microsoft-Teams-Samples:cext-trey-research-csharp-csharp
+
+provision:
+  # Set TAB_DOMAIN and TAB_ENDPOINT for local launch
+  - uses: script 
+    with:
+      run:
+        echo "::set-teamsfx-env TAB_DOMAIN=localhost:44302";
+        echo "::set-teamsfx-env TAB_ENDPOINT=https://localhost:44302";
+
+  # Creates a Teams app
+  - uses: teamsApp/create
+    with:
+      # Teams app name
+      name: cext-trey-research-csharp-${{TEAMSFX_ENV}}
+    # Write the information of created resources into environment file for
+    # the specified environment variable(s).
+    writeToEnvironmentFile: 
+      teamsAppId: TEAMS_APP_ID
+
+  # Validate using manifest schema
+  # - uses: teamsApp/validateManifest
+  #   with:
+      # Path to manifest template
+      # manifestPath: ./appPackage/manifest.json
+  # Build Teams app package with latest env value
+  - uses: teamsApp/zipAppPackage
+    with:
+      # Path to manifest template
+      manifestPath: ./appPackage/manifest.json
+      outputZipPath: ./appPackage/build/appPackage.${{TEAMSFX_ENV}}.zip
+      outputJsonPath: ./appPackage/build/manifest.${{TEAMSFX_ENV}}.json
+  # Validate app package using validation rules
```

Comment:
Since our manifest shows an error in copilotExtensions, we need to comment out the validation part. We did this by following the example in [cext-trey-research-ts](https://github.com/Jegadeesh-MSFT/Copilot-for-M365-Samples/blob/v-jegadeesh/trey-sample-csharp/samples/cext-trey-research/teamsapp.yml#L51).

![image](https://github.com/user-attachments/assets/a37a336a-bced-4d6b-989d-4d7beb88640a)

Reply:
Fixed